daq/test_modules/external_test.py

```diff
@@ -0,0 +1,153 @@
+"""Module for running docker-container tests"""
+
```

Comment:
should be external_module -- should start using the new nomenclature.

Reply:
should the ipAddrTest be a module as well?